docs/kubernetes/pouch_cri_api_annotations.md

```diff
@@ -0,0 +1,154 @@
+# CRI ANNOTATIONS SUPPORTED CHANGELOG
+
+* [Overview](#overview "Overview")
+* [The Annotations Supported](#the-annotations-supported "The Annotations Supported")
+  * [Make runtime choosing supported](#make-runtime-choosing-supported "Make runtime choosing supported")
+  * [Make lxcfs configurable supported](#make-lxcfs-configurable-supported "Make lxcfs configurable supported")
+* [Pull Request](#pull-request "Pull Request")
+
+## Overview
+
+Currently, PouchContainer has lots of advantages over other container runtimes, such as:
+
+* resource review isolation via lxcfs
+* runtime choosing for runc-based container or runv-based containers
+* and so on
+
+While actually in kubernetes, there is no related API to support these feature.
+
+While for these extended features in Kubernetes, Kubernetes has a hiden way to support this: make user-defined parameters in annotations field in pod's definition.
+When CRI manager deals the annotation details, it could pass these parameters to container manager, and container manager definitely implement these features very well.
+
+So, we need to accomplish the following things:
+
+* define the specific naming in annotations for each feature;
+* implement the transformation in CRI manager and pass them to container manager.
+
+Kubernetes Version: V1.10.0+
+
+## The Annotations Supported
+
+| Requirement                       | Field definition                             | Pull Request                               |
+|-----------------------------------|----------------------------------------------|--------------------------------------------|
+| make runtime choosing supported   | KubernetesRuntime = "io.kubernetes.runtime"  | https://github.com/alibaba/pouch/pull/1593 |
+| make lxcfs configurable supported | LxcfsEnabled = "io.kubernetes.lxcfs.enabled" | https://github.com/alibaba/pouch/pull/2210 |
+
+### Make runtime choosing supported
+
+#### What To Solve
+
+1. Support choosing the runtime for a container by making runtime choosing supported.
+
+#### How to verify it
+
+1. Prerequisites Installation, the runtime binaries you will use is necessary.
+ 
+2. You should start pouchd with the configuration like this:
+
+```
+pouchd --enable-cri --cri-version v1alpha2 --add-runtime runv=runv >pouchd.log 2>&1  &
+```
+ 
+3. After setting up your kubernetes cluster, you can create a deployment like this :
+
+```
+# cat pouch.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pouch
+  labels:
+    pouch: pouch
+spec:
+  selector:
+    matchLabels:
+      pouch: pouch
+  template:
+    metadata:
+      labels:
+        pouch: pouch
+      annotations:
+        io.kubernetes.lxcfs.enabled: "true"

```

Comment:
In this part, you said to support runtime choosing. Thus, this line is incorrect. This is about lxcfs.